vowpalwabbit/allreduce_sockets.cc

```diff
@@ -213,8 +214,7 @@ void AllReduceSockets::all_reduce_init()
       {
         if (listen(sock, kid_count) < 0)
         {
-          if (!quiet)
-            cerr << "listen: " << strerror(errno) << endl;
+          if (!quiet) cerr << "listen: " << VW::strerror_to_string(errno) << endl;
```

Comment:
should we stick to one format of if quiet in this file?

Reply:
This the clang format file at work - short blocks get collapsed to one line.

Reply:
Ill revert this line for now - I don't want to edit each one in this file

Reply:
got it, your call